place-work/v1alpha1/types.go

```diff
@@ -0,0 +1,134 @@
+	// including that it was applied and its overall status
+	PlacedManifestWork []PlacedManifestWork `json:"placedManifestWork,omitempty"`
+
+	// Summary that reflects all relevant ManifestWorks
+	PlacedManifestWorkSummary []PlacedManifestWorkSummary `json:"summary"`
+}
+
+type PlacedManifestWorkSummary struct {
+	Total       int `json:"total"`
+	Progressing int `json:"progressing"`
+	Available   int `json:"available"`
+	Degraded    int `json:"degraded"`
+	Applied     int `json:"Applied"`
+}
+
+type ManifestWorkStatus string
+
+type PlacedManifestWork struct {
+
+	// Work is an objectReference to the actual ManifestWork resource
+	Work corev1.ObjectReference `json:"workRef"`
+
+	// +kubebuilder:default=false
+	Applied bool `json:"applied"`
+
+	// ManifestWorkStatus ManifestWorkStatus.Condition can be
+	// +kubebuilder:validation:Enum=Progressing;Available;Degraded;Applied
+	// +optional
+	ManifestWorkStatus ManifestWorkStatus `json:"status"`
+
+	// Timestamp is the time when the ManifestWork was last modified
+	Timestamp string `json:"timestamp"`
+}
+
+// +kubebuilder:object:root=true
+// +kubebuilder:resource:path=placementmanifestworks,shortName=pmw;pmws,scope=Namespaced
+// +kubebuilder:storageversion
+// +kubebuilder:subresource:status
+// +kubebuilder:printcolumn:name="Placement",type="string",JSONPath=".status.conditions[?(@.type==\"PlacementVerified\")].reason",description="Reason"
+// +kubebuilder:printcolumn:name="Found",type="string",JSONPath=".status.conditions[?(@.type==\"PlacementVerified\")].status",description="Configured"
+// +kubebuilder:printcolumn:name="ManifestWorks",type="string",JSONPath=".status.conditions[?(@.type==\"ManifestworkApplied\")].reason",description="Reason"
+// +kubebuilder:printcolumn:name="Applied",type="string",JSONPath=".status.conditions[?(@.type==\"ManifestworkApplied\")].status",description="Applied"
+
+// PlaceManifestWork is the Schema for the placementmanifestworks API
+type PlaceManifestWork struct {
+	metav1.TypeMeta   `json:",inline"`
+	metav1.ObjectMeta `json:"metadata,omitempty"`
+
+	// Spec reperesents the desired ManifestWork payload and Placement reference to be reconciled
+	Spec PlaceManifestWorkSpec `json:"spec,omitempty"`
+
+	// Status represent the current status of Placing ManifestWork resources
+	Status PlaceManifestWorkStatus `json:"status,omitempty"`
+}
+
+//+kubebuilder:object:root=true
+
+// PlaceManifestWorkList contains a list of PlaceManifestWork
+type PlaceManifestWorkList struct {
+	metav1.TypeMeta `json:",inline"`
+	metav1.ListMeta `json:"metadata,omitempty"`
+	Items           []PlaceManifestWork `json:"items"`
+}
+
+type ConditionType string
+
+const (
+	PlacementDecisionVerifiedAsExpected = "PlacementDecisionVerified"
+	PlacementDecisionNotFound           = "PlacementDecisionNotFound"
+	PlacementDecisionEmpty              = "PlacementDecisionEmpty"
+
+	AsExpected = "AsExpected"
+	Processing = "Processing"
+	Partial    = "Partial"
```

Comment:
need some comments on the intention of Partial

Reply:
AsExpected is all applied.
Prcoessing is while applying or reconciling.
Partial should probably be NotAsExpected.